### PR TITLE
📄 add post: Relay Migration

### PR DESCRIPTION
**Pull request by shanewilson**

### CODE REVIEW

File: _posts/shane_wilson/2017-05-25-relay-migration.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    info: A guide to migrating a legacy Relay code base to Relay Modern
+    image: shane_wilson/relay.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+This adds a bit more detail to the [Conversion Playbook](https://facebook.github.io/relay/docs/conversion-playbook.html){:target="_blank"}
```
Comment (Fgerthoffert) [inline _posts/shane_wilson/2017-05-25-relay-migration.md]: Could you add a short intro ? Idea is to give a bit more details about the context and give readers a general idea about what the blog post is about, even if they don't know the particular technologies involved.

File: serve.sh [hunk 2]
```diff
@@ -1 +1 @@
-bundler exec jekyll serve --future
\ No newline at end of file
+bundler exec jekyll serve --future --incremental
```
Comment (cheapsteak) [inline serve.sh:1]: Was this working fine for you?
Incremental mode was buggy when I was using it, list pages would not regenerate sometimes
Notice on the site that they say it's an experimental flag

**Review by cheapsteak — APPROVED** [review]
LGTM 👍

File: serve.sh (hunk 2 above)
Comment (shanewilson) [reply serve.sh:1]: oh I think it was working but I just turned it on to see what would happen didn't mean to commit it

Comment (cheapsteak) [reply serve.sh:1]: no problems with leaving it in, we updated jekyll versions at some point, maybe they fixed it

File: _posts/shane_wilson/2017-05-25-relay-migration.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  "relay": "relay-compiler --src ./src --schema ./schema/schema.json"
+}
+```
+
+Now run the compiler - it should not find anything yet:
+
+```
+❯ yarn run relay
+yarn run v0.17.9
+$ relay-compiler --src ./src --schema ./schema/schema.json
```
Comment (Jephuff) [inline _posts/shane_wilson/2017-05-25-relay-migration.md:83]: Maybe add a note about the global watchman dependency?

Comment (shanewilson) [reply _posts/shane_wilson/2017-05-25-relay-migration.md:83]: Do you need Watchman to run it at all? or just in `--watch` mode?

Comment (Jephuff) [reply _posts/shane_wilson/2017-05-25-relay-migration.md:83]: It was throwing an error even without  `--watch`

**Pull request merged**